en/components/tabs.md

```diff
@@ -181,210 +181,151 @@ If changing the tabs' labels and icons is not enough, you can also create your o
-    data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
-</div>
+<igx-tabs>
 
-#### Using Separate router-outlet as Tabs Content
-In order to render views inside the [`igx-tabs`]({environment:angularApiUrl}/classes/igxtabscomponent.html) content, use named router outlets. In this case, implement [`onTabItemSelected`]({environment:angularApiUrl}/classes/igxtabscomponent.html#ontabitemselected) event handler to navigate and render the specified view.
+  <igx-tab-item label="Tab 1" icon="dashboard"
+		routerLink="view1"
+		routerLinkActive #rla1="routerLinkActive"
+		[isSelected]="rla1.isActive">
+	</igx-tab-item>
+
+	<igx-tab-item label="Tab 2" icon="check_circle_outline"
+		routerLink="view2"
+		routerLinkActive #rla2="routerLinkActive"
+		[isSelected]="rla2.isActive">
+	</igx-tab-item>
+
+	<igx-tab-item label="Tab 3" icon="radio_button_checked"
+		routerLink="view3"
+		routerLinkActive #rla3="routerLinkActive"
+		[isSelected]="rla3.isActive">
+	</igx-tab-item>
 
-```html
-<!-- tabs-sample-1.component.html -->
-<!-- router-outlet inside the tabs items content -->
-<igx-tabs #tabs1 (onTabItemSelected)="navigate($event)">
-  <igx-tabs-group label="Product1" name="product1">
-    <router-outlet name="product1"></router-outlet>
-  </igx-tabs-group>
-  <igx-tabs-group label="Product2" name="product2">
-    <router-outlet name="product2"></router-outlet>
-  </igx-tabs-group>
-  <igx-tabs-group label="Product3" name="product3">
-    <router-outlet name="product3"></router-outlet>
-  </igx-tabs-group>
 </igx-tabs>
 ```
 
-```typescript
-// tabs-sample-1.component.ts
-public navigate(eventArgs) {
-    const selectedIndex = eventArgs.group.index;
-    switch(selectedIndex) {
-      case 0: {
-        this.router.navigate(['/productDetails',
-          {
-            outlets:
-            {
-              product1: ['product1']
-            }
-          }
-        ]);
-        break;
-    }
-    case 1: {
-      this.router.navigate(['/productDetails',
-        {
-          outlets:
-          {
-            product2: ['product2']
-          }
-        }
-      ]);
-      break;
-    }
-    case 2: {
-      this.router.navigate(['/productDetails',
-          {
-            outlets:
-            {
-              product3: ['product3']
-            }
-          }
-        ]);
-        break;
-      }
-    }
-  }
-```
-Declare all needed route definitions that map URL path to a specific component:
+The above code creates a Tabs component with three tab items. All tab items are having the `RouterLink` directive applied which is used to specify the routing link used for the navigation. If any of these links became active, the corresponding tab item will have its `isSelected` property set because of the binding to the `RouterLinkActive` directive's `isActive` property. This way the selected tab item will always stay synchronized with the current browser's address.
```

Comment:
"If any of these links became active" - this should be changed to present simple tense (since future simple is used afterwards) -> "If any of these links becomes active"

Reply:
Addressed.